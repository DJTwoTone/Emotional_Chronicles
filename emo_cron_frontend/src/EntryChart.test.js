import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import EntryChart from './EntryChart';

it('should render', function() {
    render(
        <MemoryRouter>
            <EntryChart />
        </MemoryRouter>
    );
})

it('should match the snapshot', function() {
    const { asFragment } = render(
        <MemoryRouter>
            <EntryChart />
        </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
})