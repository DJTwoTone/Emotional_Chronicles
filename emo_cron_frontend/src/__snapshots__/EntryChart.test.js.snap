// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match the snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Emotional Radar
    </p>
    <div
      class="recharts-responsive-container"
      style="width: 75%; height: 300px;"
    />
  </div>
</DocumentFragment>
`;
